docs/reference/configuration/keywords.rst

```diff
@@ -536,7 +537,26 @@ possible to do with ERT.
 .. _run_template:
 .. topic:: RUN_TEMPLATE
 
-        Install arbitrary files in the runpath directory.
+        ``RUN_TEMPLATE`` can be used to copy files to the run path while doing magic string
+        replacement in the file content and the file name.
+
+        *Example:*
+
+        ::
+
+                ECLBASE BASE_ECL_NAME
+                RUN_TEMPLATE MY_DATA_FILE.DATA <ECLBASE>.DATA
+
+        This will copy ``MY_DATA_FILE.DATA`` into the run path and name it ``BASE_ECL_NAME.DATA``
+        while doing magic string replacement in the contents. ``%d`` can be replicated
+        by using the ``<IENS>`` magic string.
```

Comment:
What does "`%d` can be replicated" mean here?

Reply:
It was an attempt at moving people to be more explicit, but was not the best phrasing, any suggestions?

Reply:
Maybe by example:
```
Using %d in ECLBASE is supported for backwards compatibility reasons. It may be substituted with the magic substitution string <IENS>:

    ECLBASE BASE_ECL_NAME-%d
    
is equivalent to:

    ECLBASE BASE_ECL_NAME-<IENS>
```

Reply:
I wrote some more text, do you agree with the changes?

Reply:
Looks good! :100: